Sources/ClientRuntime/Retries/RetryErrorInfo.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.TimeInterval
+
+public struct RetryErrorInfo: Equatable {
```

Comment:
This protocol is defined in reference architecture.

The `isTimeout` field is added to support custom retry behavior needed by AWS SDKs.